server/api/resources/sh.postman_collection.json

```diff
@@ -0,0 +1,212 @@
+{
+	"info": {
```

Comment:
spacing seems out of place for this one. Looks like 4 spaces.

Reply:
file is auto generated from postman. Postman uses tabs. I am not aware changing postman's auto generation spacing/tab strategy